posthog/api/organization.py

```diff
@@ -133,7 +133,7 @@ def get_setup(self, instance: Organization) -> Dict[str, Union[bool, int, str, N
         }
 
 
-class OrganizationViewSet(AnalyticsDestroyModelMixin, viewsets.ModelViewSet):
+class OrganizationViewSet(viewsets.ModelViewSet):
```

Comment:
Context on why we're doing this instead of using `AnalyticsDestroyModelMixin`, doesn't look like we're changing anything.

Reply:
Because `user.current_organization` will fail after the organization is destroyed, so we need to work around it.